### PR TITLE
Oddjobs

### PR DESCRIPTION
**Pull request by Incorrect-password**
I have added some media icons, hover functions, text titles.

### CODE REVIEW

File: aboutme.html [hunk 1]
```diff
@@ -22,21 +22,26 @@
...
 				
-				<div class="flexContainer contacts">
-					<p> Lorem ipsum dolor sit amet consectetur adipisicing elit. Numquam optio repudiandae ipsam animi voluptate omnis enim aspernatur rem deleniti eveniet nam molestias vel deserunt vitae sapiente iste mollitia, eos corrupti.</p>
+				<div class="formContainer contact">
+					<h2>Contact me</h2>
+					<p> If you would like to contact me for any reason including to arrange a phone call please fill out the form below and I will get back to you as soon as possible.</p>
 					<form>
-						<input type="text-area" name="emailBody">
-						<input type="submit" value="Submit">
+						<textarea name="emailBody" form="email" placeholder ="What can I help you with?" class="form"></textarea>
```
Comment (ashleycoles) [inline aboutme.html]: Remove the space after placeholder, and remove the form attributes on these inputs, they're only needed if an input is outside of the form tags (which is hardly ever done).

File: projects.html [hunk 2]
```diff
@@ -21,60 +21,58 @@
...
 
 				</div>
 				
-				<div class="flexContainer">
-
-					<div class="tile">
+						<div class="tile">
 							<a href=#>
-								<img src=https://cdn.pixabay.com/photo/2019/08/19/07/45/pets-4415649_960_720.jpg>
+								<img src=https://cdn.pixabay.com/photo/2019/08/19/07/45/pets-4415649_960_720.jpg class="projImg">
```
Comment (ashleycoles) [inline projects.html]: All images must have an alt tag.

File: aboutme.html [hunk 3]
```diff
@@ -7,8 +7,8 @@
 		<link href="stylesheetportfolio.css" rel="stylesheet" type="text/css">
 		<title>Profile</title>
 	</head>
-		<body>
-			<nav class= "container">
+		<body class="aboutMe">
+			<nav class= "navContainer">
```
Comment (mporam) [inline aboutme.html]: theres a space after the `=`

File: aboutme.html [hunk 4]
```diff
@@ -22,21 +22,26 @@
 				</div>
 			</nav>
 			
-			<div class="container aboutMeContainer">
-				<div class="flexContainer aboutMe">
-					<div class="textBox">
-						<p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Culpa, placeat reiciendis corporis dolor minima atque illum tenetur! Ad qui sapiente, possimus facere hic laboriosam quis reiciendis dicta veritatis? Quo sint doloremque distinctio atque illo corporis laboriosam saepe, repudiandae et vitae mollitia ipsa itaque iste velit architecto. Maxime ea, assumenda aut quam perspiciatis doloribus earum, accusantium inventore illum nam voluptas eveniet?</p>
+			<div class="aboutMeContainer">
+				<div class="floatBox">
+					<div class="textBox2">
```
Comment (mporam) [inline aboutme.html]: is this a good class name

File: projects.html [hunk 5]
```diff
@@ -7,8 +7,8 @@
 		<link href="stylesheetportfolio.css" rel="stylesheet" type="text/css">
 		<title>Projects</title>
 	</head>
-		<body>
-			<nav class= "container projectContainer">
+		<body class="projects">
+			<nav class= "navContainer projectContainer">
```
Comment (mporam) [inline projects.html]: space

File: stylesheetportfolio.css [hunk 6]
```diff
@@ -9,71 +9,200 @@ html {
...
 }
 
-.aboutMe {
-	background-image: url("https://images.freeimages.com/images/small-previews/056/winter-trees-1639640.jpg");
-	object-fit: cover;
-	width: 70vw;
+.navLink:hover {
+	background-color: #837c88;
+	transform: scale(1.2);
+
```
Comment (mporam) [inline stylesheetportfolio.css]: remove empty line

**Pull request merged**